KarhooUISDK/FeatureFlags/FeatureFlagsModel.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  FeatureFlagsModel.swift
+//  KarhooUISDK
+//
+//  Created by Bartlomiej Sopala on 13/04/2023.
+//  Copyright © 2023 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+
+struct FeatureFlag: Codable {
```

Comment:
I'd suggest naming this as plural, i.e FeatureFlag**s** since this is a collection of multiple flags